src/cloudformation/vpc.template

```diff
@@ -232,37 +246,99 @@
           {
             "Key": "Name",
             "Value": {
-              "Fn::Join": ["-", [{"Ref": "AWS::StackName"}, "private"]]
+              "Fn::Join": ["-", [{"Ref": "AWS::StackName"}, "Private01"]]
             }
           }
         ]
       }
     },
+    "PrivateRouteTable02": {
```

Comment:
I dig that, will open a new issue.
I think it'd mean shifting the subnet/gateway generation logic to Python (otherwise we have NatGateway{1,2,3,4,5} to grow in us-east-1; much template waste).

This setup where N=3 _always_ is wasteful on regions with only 2 AZs (i.e. eu-central-1, ap-southeast-1). I picked 3 because the EC2 SLA kicks in when >1 AZ is down; that makes the baseline impl either available, or free (via SLA credits).